src/CodeGenerator.php

```diff
@@ -281,6 +281,16 @@ private static function generateServices(
             $code = Formatter::format($code);
             yield ["src/{$version}Gapic/{$service->gapicClientType->name}.php", $code];
 
+            // snippet gen wip
+            $snippetFiles = SnippetGenerator::generate($licenseYear, $service);
+
+            foreach ($snippetFiles as $snippetFile) {
+                $code = $snippetFile[1]->toCode();
+                $code = Formatter::format($code);
+                $methodName = Helpers::toSnakeCase($snippetFile[0]);
```

Comment:
Does it make sense to switch the return type of `generate` to `Map` with key = methodName and value = AST or whatever? 

Magic numbers aren't my favorite thing.

Reply:
I agree about the numeric index - though the generate method already returns a map instance. When iterating over the map it returns the following data structure (which I'm open to modifying, and would find beneficial unless I am missing something):

```
counter => [
    key, // as set by Map::set($key, $value)
    value
]

e.g.,

// Map::set('hello', world')

0 => ['hello', 'world']

Reply:
I changed this to an array instead of using the Map.